### DISCUSSION TITLE
Can Web3Modal SDK show how the wallet is connected to the modal (browser extension or mobile app)?

### DISCUSSION THREAD

**Discussion by lufaque** [WalletConnect/walletconnect-monorepo#4099] [Developer Support] [answered]
Hi everyone! 

Can Web3Modal SDK show how the wallet is connected to the modal (browser extension or mobile app)?

Thanks a lot for your help.

**Comment by glitch-txs** [comment]
hi, are you using Wagmi or ethers?

**Reply by lufaque** [reply]
hi! Wagmi

**Reply by lufaque** [reply]
@glitch-txs

**Reply by glitch-txs** [reply, marked as answer]
You can check the id or name of the current connected connector

**Reply by lufaque** [reply]
Thanks a lot!
